jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/SimulationServletTest.java

```diff
@@ -27,17 +27,25 @@
 import org.jbpm.designer.helper.TestHttpServletResponse;
 import org.jbpm.designer.repository.UriUtils;
 import org.jbpm.designer.web.profile.impl.JbpmProfileImpl;
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
 
 public class SimulationServletTest {
 
+    private JbpmProfileImpl profile = new JbpmProfileImpl();
```

Comment:
Could you please move this into the `setup()` method? It is more convenient to start each test with the 'clean' instance of `JbpmProfileImpl`